src/commons/sideContent/SideContentTypes.ts

```diff
@@ -37,6 +37,10 @@ export enum SideContentType {
   htmlDisplay = 'html_display'
 }
 
+export const isVisualizerTab = (tab: SideContentType) => {
+  return tab === SideContentType.substVisualizer || tab === SideContentType.envVisualizer;
+};
```

Comment:
Should the check in the `onChangeTabs` callback really be extracted out here? By the name of this function, I would expect tabs such as the data visualizer to return true when passed as argument into this function but this is not the case. You could rename this function to specifically check for the stepper and environment visualizer, but at that point you might as well inline this function in the one place it is used.

There's also the matter of future visualizers being added. If we ignore the current usage in the `onChangeTabs` callback and assume this function should do what its name suggests, it's very easy for this function to be overlooked (and not be updated) when new visualizers are added. Perhaps whether a tab is a visualizer or not can be an attribute on the `SideContentType` if we want this particular check.

Reply:
That's true. Perhaps we can just create this function locally within `Playground.tsx`, instead of exporting it. Thanks!

Reply:
Resolved in 8e2cea90de8fd233bdc7c49e6b28d10386c765a1.

Reply:
Hmm, even if the function is placed within `Playground.tsx`, wouldn't the fact that it returns false for the data visualizer still be an issue? Or at least, this is a surprising result for me.

Reply:
Hmm, that's fair. I guess this is a relatively minor this. I'll revert it back for now since it's the only thing that's blocking while we think about it.